packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2059,7 +2059,7 @@ abstract class RenderObject with DiagnosticableTreeMixin implements HitTestTarge
     assert(child._parent == this);
     assert(child.attached == attached);
     assert(child.parentData != null);
-    if (!(_isRelayoutBoundary ?? true)) {
+    if (!(child._isRelayoutBoundary ?? true)) {
```

Comment:
Unrelated change. This is a typo from https://github.com/flutter/flutter/pull/169958